test-suite/generated-src/wasm/NativeEnumUsageInterface.cpp

```diff
@@ -68,16 +68,29 @@ std::unordered_map<::testsuite::color, ::testsuite::color> NativeEnumUsageInterf
     return ::djinni::Map<::djinni_generated::NativeColor, ::djinni_generated::NativeColor>::toCpp(ret);
 }
 
-EMSCRIPTEN_BINDINGS(enum_usage_interface) {
-    em::class_<::testsuite::EnumUsageInterface>("EnumUsageInterface")
-        .smart_ptr<std::shared_ptr<::testsuite::EnumUsageInterface>>("EnumUsageInterface")
+namespace {
+    EM_JS(void, djinni_init_testsuite_enum_usage_interface, (), {
+        'testsuite'.split('.').reduce(function(path, part) {
+            if (!(part in path)) { path[part] = {}}; 
+            return path[part]}, Module);
```

Comment:
If we are going to keep the aliasing code we should format it a bit better
```suggestion
        'testsuite'.split('.').reduce(function(path, part) {
            if (!(part in path)) { path[part] = {}}; 
            return path[part]
          }, Module);
```